CONTRIBUTORS.txt

```diff
@@ -25,24 +25,29 @@ needs to be listed here.
 - Ben Cohen <ben_cohen@apple.com>
 - Bouke Haarsma <bouke@haarsma.eu>
 - Chris Eidhof <chris@eidhof.nl>
+- Christian Weinberger <christian.weinberger@me.com>
 - Cory Benfield <cbenfield@apple.com>
 - Cotton Hou <himcotton@gmail.com>
 - Damiaan Dufaux <damiaan@dpro.be>
 - Daniel Alm <Daniel.Alm@ForumD.net>
 - Daniel Dunbar <daniel_dunbar@apple.com>
 - Dario Rexin <drexin@apple.com>
+- David Evans <d.evans@apple.com>
 - David Skrundz <david.skrundz@gmail.com>
+- Emil Marashliev <emarashliev@users.noreply.github.com>
```

Comment:
@emarashliev if you tell me which email address to use, I can use a real email address instead of this odd github one.

Reply:
I use `marashliev@gmail.com` thanks for asking me! :)

Reply:
you're welcome & done